tests/test_recipe.py

```diff
@@ -3,13 +3,12 @@
 
 import pytest
 
-import helper
 from rpmlb.recipe import Recipe
 
 
 @pytest.fixture
-def ok_recipe():
-    return Recipe(helper.get_valid_recipe_file(), 'rh-ror50')
+def ok_recipe(valid_recipe_path):
+    return Recipe(str(valid_recipe_path), 'rh-ror50')
```

Comment:
It would, but since the `os.*` functions do not take PathLib object too (on Python 3.4), it is low priority issue…